packages/webapp/src/components-v2/IntegrationLogo.tsx

```diff
@@ -0,0 +1,28 @@
+import { CubeTransparentIcon, LockOpenIcon } from '@heroicons/react/24/outline';
+import { useState } from 'react';
+
+interface IntegrationLogoProps {
+    provider: string;
+    size?: number;
+}
+
+export const IntegrationLogo = ({ provider, size = 8 }: IntegrationLogoProps) => {
+    const [imgError, setImgError] = useState(false);
+
+    return (
+        <div className={`size-${size} p-1 rounded-sm flex items-center justify-center bg-white border-[0.5px] border-border-muted`}>
```

Comment:
is tailwind support any `size-N` values? Not a big deal since we seems to always use a default value but might break if truly dynamic